### PR TITLE
feat: add Actix-web

### LINKED ISSUE #6: Setup HTTP framework for the app code
look at actix: https://github.com/actix/actix-web
look at rocket: https://rocket.rs/

### PR DESCRIPTION
**Pull request by Nghondzweni**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = [ "rtc_tenclave", "rtc_uenclave", "rtc_types", "rtc_data_enclave", "rtc_data_service" ] 
+members = [ "rtc_tenclave", "rtc_uenclave", "rtc_types", "rtc_data_enclave", "rtc_data_service" , "rtc_app_backend"]
```
Comment (PiDelport) [inline Cargo.toml]: This should probably be reverted?

File: rtc_data_service/src/actix.rs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+mod actix_config{
+    use config::ConfigError;
+    use serde::Deserialize;
+
+    // Configuration specific to there server
```
Comment (PiDelport) [inline rtc_data_service/src/actix.rs]:
```suggestion
    // Configuration specific to the server
```

File: rtc_data_service/src/actix.rs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+    // App configuration
+    #[derive(Deserialize)]
+    pub struct Config {
+        pub server : ServerConfig
+    }
+
+    impl Config {
+        // Loads configuration from the evironment to the Config Struct
```
Comment (PiDelport) [inline rtc_data_service/src/actix.rs]:
```suggestion
        // Loads configuration from the environment to the Config Struct
```

File: rtc_data_service/Cargo.toml [hunk 4]
```diff
@@ -3,6 +3,7 @@ name = "rtc_data_service"
 version = "0.0.1"
 authors = ["Registree"]
 build = "build.rs"
+edition = "2018"
```
Comment (PiDelport) [inline rtc_data_service/Cargo.toml:6]: Is this the right thing?

File: Cargo.toml (hunk 1 above)
Comment (Nghondzweni) [reply Cargo.toml]: I noticed when I was building now. I made the change. I will push.

**Review by longtomjr — APPROVED** [review]
Thanks for getting us started on this side!

**Pull request merged**